### PR TITLE
bug 1668338. Fix the correct ds template

### PR DESCRIPTION
**Pull request by jcantrill**
Fix the correct DS template for fluentd to correctly resolve https://bugzilla.redhat.com/show_bug.cgi?id=1668338

### CODE REVIEW

**Comment by jcantrill** [comment]
@sdodson I think this will need manual merge

File: roles/openshift_logging_fluentd/templates/2.x/fluentd.j2 [hunk 1]
```diff
@@ -98,6 +98,8 @@ spec:
           readOnly: true
 {% endif %}
         env:
+        - name: "MERGE_JSON_LOG"
+          value: "{{ openshift_logging_fluentd_merge_json_log }}"
```
Comment (richm) [inline roles/openshift_logging_fluentd/templates/2.x/fluentd.j2]: Is there a default setting for `openshift_logging_fluentd_merge_json_log`?

Comment (sdodson) [reply roles/openshift_logging_fluentd/templates/2.x/fluentd.j2]: Yeah, seems to be set here to "true" https://github.com/openshift/openshift-ansible/blob/bae78fb8ab91348392c69777a589d3e20c8c5135/roles/openshift_logging_fluentd/defaults/main.yml#L32

**Pull request merged**